packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -510,7 +510,7 @@ const imageNodeType = ({
     },
     resolve: async (image, fieldArgs, context) => {
       const file = getNodeAndSavePathDependency(image.parent, context.path)
-      const args = { ...fieldArgs, pathPrefix }
+      const args = { ...fieldArgs }
```

Comment:
I know this can be cleaned up but it's being finicky for some reason so I'm leaving it until I have tests I can trust.